tests/loadtest/src/admin.rs

```
@@ -0,0 +1,26 @@
+use goose::prelude::*;
+
+use std::env;
+
+/// Log into the website.
+pub async fn log_in(user: &mut GooseUser) -> TransactionResult {
+    // Use ADMIN_USERNAME= to set custom admin username.
+    let admin_username = match env::var("ADMIN_USERNAME") {
+        Ok(username) => username,
+        Err(_) => "admin".to_string(),
+    };
+    // Use ADMIN_PASSWORD= to set custom admin username.
+    let admin_password = match env::var("ADMIN_PASSWORD") {
+        Ok(password) => password,
+        Err(_) => "P@ssw0rd1234".to_string(),
```

This seems like something we shouldn't include. Even if it's not a real password, probably better to just throw an error if the env vars aren't set.

---

Changed to be the export commands needed. Not sure how else to do it right now, the Err() needs to be there (tested).